index.js

```diff
@@ -1,3 +1,19 @@
-module.exports = function () {
-  return 'Hello, world';
+var lwip = require('lwip'),
+    VError = require('verror');
+var IMAGE_WIDTH = 2480;
+var IMAGE_HEIGHT = 626;
+
+module.exports = function processImage(imageData, type, callback) {
+  lwip.open(imageData, type, function(err, image) {
+    if (err) {
+      return callback(new VError(err, 'Opening image data failed.'));
+    }
+    var colour = image.getPixel(0,0);
+    image.batch().
```

Comment:
I'm still not entirely sold on this practise.  I know it is better for parsers checking for syntax errors... jshint doesn't like it and nolonger has the option to ignore it :(...  but it is a little harder to group together visually.  Can we at least indent the following lines a little so that it is more obvious that they aren't some more function calls (without having to check the previous line for a trailing .)